doc/changelog.rst

```diff
@@ -3,6 +3,7 @@
 Release Notes
 =============
 
+- :bug:`common` _VITE_IGNORE -> VITE_IGNORE: django settings variable can't start with _.
```

Comment:
Just a heads-up: I'll remove this line from the release notes, as I think that it will only confuse the readers needlessly: the reader of the release notes is assumed to be an interested pretalx user or administrator of a self-hosted instance, and this note would serve to confuse them, without having any impact on them.